### PR TITLE
More Sites Updates

### PR DESCRIPTION
**Pull request by kshitij9896**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5263"></a>

### CODE REVIEW

File: data/kshitij9896.json [hunk 1]
```diff
@@ -6,39 +6,51 @@
...
+      "group":"Services",
+      "name": "Can Help You With",
+      "url": "https://thisiskshitijbansod.netlify.app/",
+      "icon": "FaGlobe",
+    },
+    {
+      "group":"Services",
+      "name": "Bussiness Queries | For Hiring ",
+      "url": "mailto:contactkshitij9@gmail.com",
+      "icon": "SiGmail",
```
Comment (priyankarpal) [inline data/kshitij9896.json:23]: Remove last comma 
```suggestion
      "icon": "SiGmail"
```

File: data/kshitij9896.json [hunk 2]
```diff
@@ -6,39 +6,51 @@
...
+    { "icon": "FaDev", "url" : "https://dev.to/kshitij9896"},
+    { "icon": "FaBlog", "url" : "https://www.showwcase.com/thisiskshitijbansod"}
   ],
   "links": [
     {
-      "group": "Socials",
+      "group":"Services",
+      "name": "Can Help You With",
+      "url": "https://thisiskshitijbansod.netlify.app/",
+      "icon": "FaGlobe",
```
Comment (priyankarpal) [inline data/kshitij9896.json:17]: same here remove last comma  
```suggestion
      "icon": "FaGlobe"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
One Suggestion...
- when you create a PR please add some description & screenshots it helps us to understand the changes quickly.

**Comment by kumarsonsoff3** [comment]
Merged the one with latest changes in the same file #5264

**Comment by kshitij9896** [comment]
> Merged the one with latest changes in the same file #5264

Thank You

**Pull request closed without merging**